ecms_base/modules/custom/ecms_distribution/ecms_distribution.module

```diff
@@ -0,0 +1,253 @@
+<?php
+
+/**
+ * @file
+ * ecms_distribution.module
+ */
+
+declare(strict_types=1);
+
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\ReplaceCommand;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Get the social configuration from pattern lab if available.
+ *
+ * @return array|null
+ *   The available social navigation options, or null if an error.
+ */
+function get_patternlab_social_source(): ?array {
+  $return = [];
+  /** @var \Drupal\Core\Extension\ThemeHandler $themeHandler */
+  $themeHandler = \Drupal::service('theme_handler');
+
+  $defaultTheme = $themeHandler->getDefault();
+
+  $path = drupal_get_path('theme', $defaultTheme);
+
+  // If the source file doesn't exist, ignore the form alteration.
+  if (!file_exists("{$path}/ecms_patternlab/source/_data/social-config.json")) {
+    return NULL;
+  }
+
+  $json = json_decode(file_get_contents("{$path}/ecms_patternlab/source/_data/social-config.json"));
+
+  if (!property_exists($json, 'social-platforms')) {
+    return NULL;
+  }
+
+  foreach ($json->{'social-platforms'} as $key => $object) {
+    $return[$key] = $object->{'link-title'};
+  }
+
+  $return['other'] = t('Other');
+
+  return $return;
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter() for menu_link_content_menu_link_content_form.
+ */
+function ecms_distribution_form_menu_link_content_menu_link_content_form_alter(&$form, FormStateInterface $form_state) {
```

Comment:
If you can add the argument and return type here let's do that.

Reply:
I added the missing return type hint.

The `ecms_distribution` module was added in the 9017 update here: 
https://github.com/State-of-Rhode-Island-eCMS/ecms_profile/blob/f1309acea2e2e36d36e33853e6fe7b9ae2e7204c/ecms_base/ecms_base.profile#L113

Reply:
AH see it now!